app/search/classification/domain.py

```diff
@@ -0,0 +1,23 @@
+from urllib.parse import urlparse
+
+from .base import BaseClassifier
+
+
+class DomainClassifier(BaseClassifier):
+    """
+    Classifier that is applied if the returned result is the root address of a
+    domain name. Adds:
+
+    fqdn - the fully qualified domain name
+    logo - the URL to a logo, as determined by Clearbit.
```

Comment:
After staring at the classifier code here and in the other open PRs, I think we're fine to leave this as-is. I don't think we'll get much value from separating out the API interaction and the transformation of the results, since there's currently almost no transformation logic.